server/views/pages/goal/add-note/index.njk

```diff
@@ -42,17 +42,20 @@
         <div class="govuk-form-group">
           <input type="hidden" name="_csrf" value="{{ csrfToken }}" />
 
-          {{ govukTextarea({
-            name: "note",
-            id: "note",
-            value: form.note,
-            label: {
-              text: "Add a note to this goal (optional)",
-              classes: "govuk-label--l",
-              isPageHeading: true
-            },
-            errorMessage: errors | findError('note')
-          }) }}
+          <div class="govuk-form-group">
+            <h1 class="govuk-label-wrapper">
```

Comment:
Need to use vanilla HTML here, rather than Nunjucks, because thats the only way we can use the GOVUK warning text component alongside the Textarea component